src/code/bank0.asm

```diff
@@ -6881,37 +6881,48 @@ data_36B0::
     db   $43, $44                                 ; $36B0
 
 LoadObject_OpenDoorTop::
-    ld   a, $04                                   ; $36B2: $3E $04
-    call label_36C4                               ; $36B4: $CD $C4 $36
+    ld   a, ROOM_STATUS_DOOR_OPENED               ; $36B2: $3E $04
+    call addIndoorRoomStatus                               ; $36B4: $CD $C4 $36
```

Comment:
Nitpick: in this project function names are Capitalized.

Also, I understand where the `add` comes from in the name (status is updated, not overwritten), but I feel like `add` looks a bit like numbers being added together (where here statuses are more `or`-ed together. Would `UpdateIndoorRoomStatus` be a more suitable name?

Reply:
I agree. This name would be better.